mk-local-include.sh

```diff
@@ -0,0 +1,89 @@
+#!/bin/bash
+
+# Binutils provide certain headers that are clashing with system-wide
+# headers on Cygwin and MacOS. They do so in different manners, though,
+# so it is not possible to just tweak the inclusion/library order.
+# Hence, we prepare a filtered inclusion directory where we only place
+# the headers that we need for BFD/binutils. To determine them, we'll
```

Comment:
I thought letting the compiler figure out all the dependencies of a given header would be less prone to breakage if this list changes in the new binutils version. But since you're dependent on this already, adding one more dependency isn't a big deal.

I'll update the patch in the evening.